src/languages/singlestoredb/singlestoredb.functions.ts

```diff
@@ -0,0 +1,302 @@
+import { flatKeywordList } from '../../utils';
+
+export const functions = flatKeywordList({
+  // https://docs.singlestore.com/managed-service/en/reference/sql-reference.html
+  // https://docs.singlestore.com/managed-service/en/reference/sql-reference/restricted-keywords/list-of-restricted-keywords.html
```

Comment:
There's no list of function names at this URL. Well, I presume the list of keywords also includes all function names, but it's not really of much help.

Reply:
Here is the list of all the functions that are documented:
* https://docs.singlestore.com/managed-service/en/reference/sql-reference/vector-functions/vector-functions.html
* https://docs.singlestore.com/managed-service/en/reference/sql-reference/window-functions/window-functions.html
* https://docs.singlestore.com/managed-service/en/reference/sql-reference/string-functions/string-functions.html
* https://docs.singlestore.com/managed-service/en/reference/sql-reference/conditional-functions/conditional-functions.html
* https://docs.singlestore.com/managed-service/en/reference/sql-reference/numeric-functions/numeric-functions.html
* https://docs.singlestore.com/managed-service/en/reference/sql-reference/geospatial-functions/geospatial-functions.html
* https://docs.singlestore.com/managed-service/en/reference/sql-reference/json-functions/json-functions.html
* https://docs.singlestore.com/managed-service/en/reference/sql-reference/information-functions/information-functions.html
* https://docs.singlestore.com/managed-service/en/reference/sql-reference/aggregate-functions/aggregate-functions.html
* https://docs.singlestore.com/managed-service/en/reference/sql-reference/time-series-functions/time-series-functions.html
* https://docs.singlestore.com/managed-service/en/reference/sql-reference/identifier-generation-functions.html
* https://docs.singlestore.com/managed-service/en/reference/sql-reference/date-and-time-functions/date-and-time-functions.html
* https://docs.singlestore.com/managed-service/en/reference/sql-reference/distinct-count-estimation-functions.html
* https://docs.singlestore.com/managed-service/en/reference/sql-reference/full-text-search-functions/full-text-search-functions.html
* https://docs.singlestore.com/managed-service/en/reference/sql-reference/regular-expression-functions.html